README.md

```diff
@@ -138,6 +138,28 @@ end
 [ Info: Yo Dawg! it is all good
 ```
 
+### Respecting `maxlog` convention
+
+An `ActiveFilterLogger` can be used to wrap another logger to obey `maxlog` directives, for example,
+similar to the `make_throttled_logger` example below,
+```julia
+function make_maxlog_logger(logger)
+    counts = Dict{Symbol,Int}()
```

Comment:
I think this needs to be
```suggestion
    counts = Dict{Any,Int}()
```
see e.g. https://github.com/JuliaLang/julia/pull/40820

Reply:
annoying! ok, thanks